hail/python/hail/expr/builders.py

```diff
@@ -83,7 +83,7 @@ def f(base):
     @typecheck_method(value=expr_any, then=expr_any)
     def when(self, value, then) -> 'SwitchBuilder':
         """Add a value test. If the `base` expression is equal to `value`, then
-         returns `then`.
+        returns `then`.
```

Comment:
Unrelated formatting fix for [SwitchBuilder.when docs](https://hail.is/docs/0.2/functions/hail.expr.builders.SwitchBuilder.html#hail.expr.builders.SwitchBuilder.when).